src/blog/2023/03/3-quick-node-red-tips-4.md

```diff
@@ -0,0 +1,52 @@
+```
+
+### 2. Perform simple maths functions using JSONata in Change nodes
```

Comment:
```suggestion
### 2. Perform simple maths functions using JSON Data in Change nodes
```

Reply:
[JSONata](https://docs.jsonata.org/) is the correct name.